apps/backend/src/datasources/stfc/StfcProposalDataSource.ts

```diff
@@ -134,4 +137,64 @@ export default class StfcProposalDataSource extends PostgresProposalDataSource {
 
     return result;
   }
+
+  async getProposalsFromView(
+    filter?: ProposalsFilter,
+    first?: number,
+    offset?: number,
+    sortField?: string,
+    sortDirection?: string,
+    searchText?: string
+  ): Promise<{ totalCount: number; proposalViews: ProposalView[] }> {
+    const proposals = await super.getProposalsFromView(
+      filter,
+      first,
+      offset,
+      sortField,
+      sortDirection,
+      searchText
+    );
+
+    const technicalReviewers = new Set(
+      proposals.proposalViews
+        .filter((proposal) => !!proposal.technicalReviewAssigneeId)
+        .map((proposal) => proposal.technicalReviewAssigneeId.toString())
+    );
+
+    const technicalReviewersDetails =
+      await stfcUserDataSource.getStfcBasicPeopleByUserNumbers(
```

Comment:
I guess this will slow down the proposal view query a bit but maybe not that much. We can have better overview once it gets tested properly with real data.

Reply:
Hopefully there should not be more than 20-40 different reviewers so it shouldn't slow it down too much, but we will see. I don't really see another way of populating that field since we only store user numbers in proposals db.